asf_lifetime_cost_model/analysis/notebooks/cost_computations_demo.py

```diff
@@ -0,0 +1,106 @@
+# %%
+from asf_lifetime_cost_model.pipeline import cost_computations
+
+# %% [markdown]
+# The cost of installing an air source heat pump, subtracting the subsidy from the upfront cost:
+
+# %%
+cost_computations.compute_upfront_cost(
+    heating_system="ashp",
+    archetype="pre_1950_semi_terraced_house_5_rooms",
+    annual_cost_reduction=0.01,
+    installation_year=2026,
+    decile=50,
+    subsidy_model_or_value="flat",
+)
+
+# %% [markdown]
+# The cost of installing an air source heat pump, WITHOUT subtracting the subsidy from the upfront cost:
+
+# %%
+cost_computations.compute_upfront_cost(
+    heating_system="ashp",
+    archetype="pre_1950_semi_terraced_house_5_rooms",
+    annual_cost_reduction=0.01,
+    installation_year=2026,
+    decile=50,
+    subsidy_model_or_value="flat",
```

Comment:
```suggestion
    subsidy_model_or_value=0,
```

I think you forgot to adjust the arguments here to demonstrate the case without the subsidy. I'm assuming setting the `subsidy_model_or_value` to 0 is the only way to exclude the subsidy?

Reply:
For no subsidy you can use "no subsidy". I actually need to change this to take a dictionary of subsidy values per year rather than just one value.